index.js

```diff
@@ -0,0 +1,749 @@
+/*!***************************************************
+ * node-diacritics-transliterator
+ * http://diacritics.io/
+ * Copyright (c) 2016, Julian Motz & Rob Garrison
+ * Released under the MIT license https://git.io/v1EBe
+ *****************************************************/
+"use strict";
+
+const databaseURL = "http://api.diacritics.io/",
+
+    request = require("sync-request"),
+    regenerate = require("regenerate");
+
+/**
+ * Support functions
+ * @access protected
+ */
+/**
+ * Get current API version from diacritics/api package.json; sets
+ * `module.exports.currentVersion` and `module.exports.maxVersion`
+ * (https://github.com/diacritics/api/raw/master/package.json)
+ * @access protected
+ */
+function getCurrentVersion() {
+    let current = 1;
+    const pkg = getJSON("https://git.io/v1Ejh");
+    if(pkg && pkg.version) {
+        const major = parseInt(pkg.version.split(".")[0], 10);
+        // beta testing major version = 0
+        current = major > 1 ? major : 1;
+    }
+    module.exports.currentVersion = "v" + current;
+    // make readonly?... modified in test.js for testing
```

Comment:
It's a reminder since we were discussing removing the ability to set versions... I'll go ahead and remove it.